ui/client/components/graph/node-modal/editors/expression/EditorType.ts

```diff
@@ -27,6 +28,7 @@ type EditorConfig = {
   switchableTo?: (expressionObj: ExpressionObj, param?: ParamType, values?: ValuesType) => boolean,
   values?: (param: ParamType, values: ValuesType) => $TodoType,
   switchable?: Function,
+  validators?: Function,
```

Comment:
if you know this function args and return type, provide type.

Reply:
Done